drv/Maps.drv

```diff
@@ -91,23 +98,12 @@ public class MAPS {
 #endif
 
 		@Override
-		public void putAll(final Map<? extends KEY_GENERIC_CLASS, ? extends VALUE_GENERIC_CLASS> m) { throw new UnsupportedOperationException(); }
+		public void putAll(Map<? extends KEY_GENERIC_CLASS, ? extends VALUE_GENERIC_CLASS> m) { throw new UnsupportedOperationException(); }
```

Comment:
Definitely. I still am a supporter of `final`ing everything so I just did that ;)